pkgs/cli/test/integration.zig

```diff
@@ -39,7 +39,7 @@ fn getZeamExecutable() ![]const u8 {
 /// Returns the process handle for cleanup, or error if startup fails
 fn spinBeamSimNode(allocator: std.mem.Allocator, exe_path: []const u8) !*process.Child {
     // Set up process with beam command and mock network
-    const args = [_][]const u8{ exe_path, "beam", "--mockNetwork", "true" };
+    const args = [_][]const u8{ exe_path, "beam", "--mockNetwork", "true", "--num_validators", "4" };
```

Comment:
no need to change num validators, the default is 3, and 2 validators (one for each node) are able to justify and finalize